common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4673,6 +4673,41 @@
     },
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_SHOW_CREATE_TABLE" : {
+    "message" : [
+      "Unsupported show create table command:"
+    ],
+    "subClass" : {
+      "ON_SPARK_DATA_SOURCE_TABLE_WITH_AS_SERDE" : {
+        "message" : [
+          "<table> is a Spark data source table. Use `SHOW CREATE TABLE` without `AS SERDE` instead."
+        ]
+      },
+      "ON_TEMPORARY_VIEW" : {
+        "message" : [
+          "SHOW CREATE TABLE is not supported on a temporary view: <table>."
+        ]
+      },
+      "ON_TRANSACTIONAL_HIVE_TABLE" : {
+        "message" : [
+          "SHOW CREATE TABLE doesn't support transactional Hive table. Please use `SHOW CREATE TABLE <table> AS SERDE` to show Hive DDL instead."
+        ]
+      },
+      "WITH_UNSUPPORTED_FEATURE" : {
+        "message" : [
+          "Failed to execute SHOW CREATE TABLE against table/view <table>, which is created by Hive and uses the following unsupported feature(s)",
+          "<unsupportedFeatures>"
+        ]
+      },
+      "WITH_UNSUPPORTED_SERDE_CONFIGURATION" : {
```

Comment:
I think 'WITH_UNSUPPORTED_CONFIG' may be more general, not necessarily limited to seder config, right?

On the other hand, I'm not sure if it's suitable to further merge WITH_UNSUPPORTED_FEATURE`` and `WITH_UNSUPPORTED_SERDE_CONFIGURATION` to make it more general."